code/__defines/__compile_options.dm

```diff
@@ -29,3 +29,8 @@
 #endif //ifdef GC_FAILURE_HARD_LOOKUP
 
 #endif //ifdef REFERENCE_TRACKING
+
+
+#define BITSHIFT_LEFT(X, N) (X << (N))
+#define BITSHIFT_RIGHT(X, N) (X >> (N))
```

Comment:
This is unnecessary to have, and it shouldn't be in this file anyways.

Reply:
it will be required when we shift to BITFLAG()

Reply:
@tichys we should not be using a macro for bitflags, and neither does TG so what's the point